qiita_db/study.py

```diff
@@ -340,6 +360,17 @@ def create(cls, owner, title, efo, info, investigation=None):
                "efo_id) VALUES (%s, %s)".format(cls._table))
         conn_handler.executemany(sql, [(study_id, e) for e in efo])
 
+        # insert portal into database
+        portal_id = convert_to_id(qiita_config.portal, 'portal_type', 'portal')
+        portal_sql = ""
+        if qiita_config.portal != "QIITA":
```

Comment:
Do we really need to add studies to qiita? I think we do not need to add portal qiita as we should be able to select every study, right?